backend/internal/server/server.go

```diff
@@ -12,6 +12,7 @@ import (
 	// Blank imports for apis to cause init functions to run
 	_ "github.com/jak103/powerplay/internal/server/apis/auth"
 	_ "github.com/jak103/powerplay/internal/server/apis/chat"
+	_ "github.com/jak103/powerplay/internal/server/apis/groups"
```

Comment:
I believe this folder is added a second time in the imports for the file. Line 20 can be deleted so the imports are in alphabetical order.

Reply:
I see what you're saying after I merged the latest from main in. Updated.